docs/content/menu/index.md

```diff
@@ -57,6 +57,7 @@ headless: true
     - [Custom Gradle Plugins]({{< ref "/ci/CustomGradlePlugins.md" >}})
     - [Impact analysis]({{< ref "/ci/ImpactAnalysis.md" >}})
     - [Containers]({{< ref "/ci/Containers.md" >}})
+    - [Lint workers API workaround]({{< ref "/ci/LintWorkerApiWorkaround.md">}})
```

Comment:
Such hacks shouldn't be on the second level of menu

Reply:
i though about it too
should we introduce third level?)